workflow-scripts/update_lambda_template.py

```diff
@@ -0,0 +1,22 @@
+from ruamel.yaml import YAML
+
+yaml = YAML()
+
+file_path = 'lambda-template.yaml'
+
+with open(file_path, 'r') as file:
+    data = yaml.load(file)
```

Comment:
Can we add error handling to validate if at all the file is missing

Reply:
@voorepreethi updated and validated. I have added details to the pr description. Also refer to test pr for further validation